indexer/services/comlink/src/controllers/api/v4/compliance-controller.ts

```diff
@@ -81,4 +124,23 @@ router.get(
   },
 );
 
+async function checkRateLimit(ipAddress: string) {
+  try {
+    await Promise.all([
+      screenProviderLimiter.consume(ipAddress, uncachedQueryPoints),
+      screenProviderGlobalLimiter.consume(globalRateLimitKey, uncachedQueryPoints),
```

Comment:
iiuc,  `screenProviderLimiter` is ensuring a single IP address doesn't make this request too often and GlobalLimiter is ensuring that this endpoint overall doesn't get used too often right? Can we document this in the code? EIther here or in  `rate-limiter.ts`

Reply:
Added comments to `rate-limiter.ts`.